app/server.js

```diff
@@ -1,12 +1,12 @@
 "use strict";
 
-const http = require("http"),
+const https = require("https"),
   express = require("express");
 
 const ServerFactory = function(config, logger) {
 
   const app = express(),
-    server =  http.createServer(app);
```

Comment:
@tejohnso We are, but I was just considering this as a safe feature to ensure a request with http won't fail in case we miss something.